docs/root/configuration/other_protocols/thrift_filters/router_filter.rst

```diff
@@ -23,3 +23,9 @@ The filter outputs statistics in the *thrift.<stat_prefix>.* namespace.
   unknown_cluster, Counter, Total requests with a route that has an unknown cluster.
   upstream_rq_maintenance_mode, Counter, Total requests with a destination cluster in maintenance mode.
   no_healthy_upstream, Counter, Total requests with no healthy upstream endpoints available.
+  request_call, Counter, Total requests with the "Call" message type.
+  request_oneway, Counter, Total requests with the "Oneway" message type.
+  request_invalid_type, Counter, Total requests with an unsupported message type.
+  response_reply, Counter, Total responses with the "Reply" message type. Includes both successes and errors.
+  response_exception, Counter, Total responses with the "Exception" message type.
+  response_invalid_type, Counter, Total responses with an unsupported message type.
```

Comment:
Hmm the new requests would be under a different prefix though (e.g.: the cluster name)?